app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,21 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level

```

Comment:
The attribute name 'hummer_level' seems to be a typo. It should likely be 'hammer_level' to reflect the intended meaning.

Reply:
Of course, I wanted to write 'hammer_level', but in task I read, that I need to add the parameter 'hummer_level'. In tests this parameter was also written as 'hummer_level'.